applications/Preprocessor/lib/mesh/MeshImpl/Mesh.h

```diff
@@ -261,7 +292,14 @@ class Mesh {
     /// localIndex of the Element.
     template <std::size_t d>
     bool checkEntities(tag<d> dimTag) const {
```

Comment:
Can we give this a more descriptive name?

Reply:
Not sure, as it performs two checks:

 1. Check that the id of the entity corresponds to its index in the list of entities
 2. Check that if the MeshEntity has (element, localIndex) as bounding element, that the element has this entity at localIndex.

So I'm not really sure on how to provide a better name. Maybe if we split these checks into two functions.

Reply:
Sounds like a good idea.